runtime/integration-tests/src/generic/envs/fudge_env.rs

```diff
@@ -0,0 +1 @@
+// TODO: implement generic::env::Env for an environment using fudge
```

Comment:
Do we still want to do this in this PR?

Reply:
No, this will be a future PR not to block this. That file is just a placeholder.

Reply:
I should create an issue for this

Reply:
https://github.com/centrifuge/centrifuge-chain/issues/1586